app/features/onboarding/onboarding_controller.rb

```diff
@@ -139,6 +141,8 @@ class OnboardingController < ControllerBase
 
         scope_validation_error = FastlaneCI::GitHubService.token_scope_validation_error(params[:clone_user_api_token])
         if scope_validation_error.nil?
+          validate_api_token_correct(params[:clone_user_api_token].strip, "initial_clone_user")
```

Comment:
`clone_user_api_token` and `initial_clone_user `
I think this should be the bot stuff now (when doing the invitation dance)

Reply:
Sorry, I don't really follow. Could you please clarify?

Reply:
we aren't really clear about the purpose of `params[:clone_user_api_token]`. I was thinking we no longer needed it because we should be cloning the config repo using the bot's api_token, instead of using a different `clone_user_api_token`. 

We can do that clone once the bot is invited to the repo and the bot accepts the invitation too. That's the invitation dance I was referring to 👯‍♂️👯

Reply:
But we still need to validate the token that is used to invite the bot user is 40 characters, and has the correct scopes - right?

Reply:
Yup 💯

Reply:
I think the confusing thing right now is the naming of `clone_user_api_token`. Should we name it something like - `REPO_USER_API_TOKEN` (probably not the best name, but just throwing a suggestion out there).

Reply:
Could this potentially be done in another PR - so that we have the initial fix for the onboarding issues merged ASAP for alpha users, and then clarify the nomenclature at a later time? 
Thoughts?

Reply:
Maybe even `INITIAL_ONBOARDING_USER_API_TOKEN`, thoughts?

Reply:
Yeah, good idea 👍

Reply:
I like that idea. I'll make the changes to this PR when I get off work this evening.